modules/incentives/src/lib.rs

```diff
@@ -328,10 +295,39 @@ pub mod module {
 				0
 			}
 		}
+
+		fn on_runtime_upgrade() -> Weight {
```

Comment:
we don't want this anymore right

Reply:
But other storages still need to be migrated on `on_runtime_upgrade`.
Do you mean to add calls to trigger manual migration for all storage like `migrate_share_and_withdrawn_reward` and `migrate_pending_rewards`?

Reply:
right. that's fine then.